static/bidder-info/nexx360.yaml

```diff
@@ -0,0 +1,19 @@
+endpoint: "http://fast.nexx360.io/prebid-server"
```

Comment:
<img width="1707" alt="Screenshot 2025-04-18 at 12 51 17 PM" src="https://github.com/user-attachments/assets/de9559fc-f893-4ecb-ab7c-fb51ad314a20" />

Reply:
This is linked to our firewall policy which restrains access to specific countries. I've added a special rule to skip it for your tests

Reply:
Still having same error

Reply:
Indeed, we had some issues with rule, fixed now